### ISSUE TITLE
error CDS0001

### ISSUE THREAD

**Issue by Blacksheep13-NJ** [hougaard/Youtube-Video-Sources#1]
Hi Hougaard, 

I hope this message finds you well. I'm getting Authenticating CDS using ServiceUri=https://******.crm6.dynamics.com
error CDS0001: 

I'm running 
PS F:\ALProjectDataverse> C:\users\"user"\.vscode\extensions\ms-dynamics-smb.al-7.2.475273\bin\altpgen.exe -project:F:\ALProjectDataverse -packagecachepath:F:\ALProjectDataverse\.alpackages -serviceuri:https://******.crm6.dynamics.com -entities:cr55c_job_test -baseid:50000 -tabletype:CDS

I hope you can give some insights...thanks much :)

**Comment by hougaard** [comment]
Do you have MFA active on the user you're using?

**Comment by Blacksheep13-NJ** [comment]
yes, I do get that and authenticate with it.. could that be a problem? should I turn off MFA

**Comment by hougaard** [comment]
Worth a try...

**Comment by Blacksheep13-NJ** [comment]
PS F:\ALProjectDataverse> C:\users\"user"\.vscode\extensions\ms-dynamics-smb.al-7.2.475273\bin\altpgen.exe -project:F:\ALProjectDataverse -packagecachepath:F:\ALProjectDataverse\.alpackages -serviceuri:https://******.crm6.dynamics.com -entities:cr55c_job_test -baseid:50000 -tabletype:CDS

Microsoft (R) AL Table Proxy Generator version 7.2.7.16535
Copyright (C) Microsoft Corporation. All rights reserved

Authenticating CDS using ServiceUri=https://******.crm6.dynamics.com
error CDS0001: 


no joy :(

**Comment by ALGoWithNishant** [comment]
Hi Erik , 

I hope you are doing fine , i am getting below error while authenticating Dataverse Script in powershell.

Can you please provide your valuable inputs ?

Script - 
C:\Users\nisha\Documents\AL\CRMDemo> \Users\nisha\.vscode\extensions\ms-dynamics-smb.al-11.0.827766\bin\altpgen.exe
 -Project:"C:\Users\nisha\Documents\AL\CRMDemo" 
-packagecachepath:"C:\Users\nisha\Documents\AL\CRMDemo\.vscode\.alcache" 
-Serviceuri:"https://orgeb8d3f33.crm.dynamics.com"
-Entities:Contact,Account 
-BaseId:50502
-TableType:CDS

**Error :-** 
Authenticating Dataverse using ServiceUri=https://orgeb8d3f33.crm.dynamics.com
Authenticated.
Retrieving entity metadata.
System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Dynamics.Nav.TableProxyGenerator.BuildContext.<>c__DisplayClass25_0.<.ctor>b__3(SourceTableExtensionTypeSymbol ao) in D:\a\_work\1\s\source\Prod\Microsoft
.Dynamics.Nav.TableProxyGenerator\BuildContext.cs:line 107
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToArray()
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at System.Collections.Immutable.ImmutableArray.CreateRange[T](IEnumerable`1 items)
   at Microsoft.Dynamics.Nav.CodeAnalysis.ImmutableArrayExtensions.ToImmutableArrayOrEmpty[T](IEnumerable`1 items) in D:\a\_work\1\s\source\Prod\Microsoft.Dynamics.Nav.Co
deAnalysis\Collections\ImmutableArrayExtensions.cs:line 30
   at Microsoft.Dynamics.Nav.TableProxyGenerator.BuildContext..ctor(Compilation compilation, XrmMetadataService service, ImmutableArray`1 entityNames, Nullable`1 tableTyp
e, String prefix, Int32 baseId, Boolean test) in D:\a\_work\1\s\source\Prod\Microsoft.Dynamics.Nav.TableProxyGenerator\BuildContext.cs:line 105
   at Microsoft.Dynamics.Nav.TableProxyGenerator.TableProxyGeneratorTool.RunCore(CompilerTextWriter consoleOutput, ErrorLogger errorLogger, CancellationToken cancellation
Token) in D:\a\_work\1\s\source\Prod\Microsoft.Dynamics.Nav.TableProxyGenerator\TableProxyGeneratorTool.cs:line 47
   at Microsoft.Dynamics.Nav.CodeAnalysis.CompilationDependentTool.Run(CompilerTextWriter consoleOutput, ErrorLogger errorLogger, CancellationToken cancellationToken) in 
D:\a\_work\1\s\source\Prod\Microsoft.Dynamics.Nav.TableProxyGenerator\CompilationDependentTool.cs:line 125
   at Microsoft.Dynamics.Nav.CodeAnalysis.CompilationDependentTool.Run(CompilerTextWriter consoleOutput, CancellationToken cancellationToken) in D:\a\_work\1\s\source\Pro
d\Microsoft.Dynamics.Nav.TableProxyGenerator\CompilationDependentTool.cs:line 77
   at Microsoft.Dynamics.Nav.TableProxyGenerator.TableProxyGeneratorCommandLineTool.Run(String clientDirectory, String baseDirectory, String[] args) in D:\a\_work\1\s\sou
rce\Prod\altpgen\TableProxyGeneratorCommandLineTool.cs:line 29


Thanks in Advance!
